lib/matplotlib/widgets.py

```diff
@@ -809,6 +809,24 @@ def on_changed(self, func):
         """
         return self._observers.connect('changed', lambda val: func(val))
 
+    def update_range(self, vmin=None, vmax=None):
+        """Update the range of the slider"""
+        if not vmin and not vmax:
```

Comment:
```suggestion
        if vmin is None and vmax is None:
```
We need to be explicit, because we don't want to catch `vmin=0`.

Reply:
Done